scripts/train_agents.py

```diff
@@ -197,52 +232,64 @@ def N_1_FCP(args):
     fcp_eval_types = {'generate' : [], 'load': []}
     fcp_curriculum = Curriculum(train_types=fcp_train_types, is_random=True)
 
-    primary_train_types = [TeamType.SELF_PLAY_LOW, TeamType.SELF_PLAY_MEDIUM, TeamType.SELF_PLAY_HIGH]
-    primary_eval_types = {'generate': [TeamType.SELF_PLAY_LOW, TeamType.SELF_PLAY_MEDIUM, TeamType.SELF_PLAY_HIGH],
-                                'load': []}
+    primary_train_types = [
+        TeamType.SELF_PLAY_LOW,
+        TeamType.SELF_PLAY_MEDIUM,
+        TeamType.SELF_PLAY_HIGH
+    ]
+    primary_eval_types = {
+        'generate': [
+            TeamType.SELF_PLAY_LOW,
+            TeamType.SELF_PLAY_MEDIUM,
+            TeamType.SELF_PLAY_HIGH
+        ],
+        'load': []
+    }
     n_1_fcp_curriculum = Curriculum(train_types=primary_train_types, is_random=True)
 
-    get_N_X_FCP_agents(args=args,
-                        fcp_train_types=fcp_curriculum.train_types,
-                        fcp_eval_types=fcp_eval_types,
-                        n_1_fcp_train_types=n_1_fcp_curriculum.train_types,
-                        n_1_fcp_eval_types=primary_eval_types,
-                        fcp_curriculum=fcp_curriculum,
-                        n_1_fcp_curriculum=n_1_fcp_curriculum,
-                        unseen_teammates_len=unseen_teammates_len)
+    get_N_X_FCP_agents(
+        args=args,
+        fcp_train_types=fcp_curriculum.train_types,
+        fcp_eval_types=fcp_eval_types,
+        n_1_fcp_train_types=n_1_fcp_curriculum.train_types,
+        n_1_fcp_eval_types=primary_eval_types,
+        fcp_curriculum=fcp_curriculum,
+        n_1_fcp_curriculum=n_1_fcp_curriculum,
+        unseen_teammates_len=unseen_teammates_len
+    )
 
 
 def set_input(args):
     args.num_players = args.teammates_len + 1
 
     two_chefs_layouts = [
+        'coordination_ring',
```

Comment:
Could you please separate standard layouts from aamas layouts:

aamas layouts = [selected_ ..., ]
standard layouts = [...]

\# two_chefs_layouts = standard_layouts 
two_chefs_layouts = aamas_layouts

Reply:
Yes, I will define special layouts and standard layouts for that. 

But I assume we still train agents for both instead of setting up a separate training.